query-algebrizer/src/clauses/mod.rs

```diff
@@ -770,18 +771,20 @@ impl ConjoiningClauses {
             .ok()
     }
 
-    fn get_attribute_for_value<'s>(&self, schema: &'s Schema, value: &TypedValue) -> Option<&'s Attribute> {
+    fn get_attribute_for_value<'s>(&self, schema: &'s Schema, value: &TypedValue) -> Option<(&'s Attribute, KnownEntid)> {
```

Comment:
Looks like it might be an orphaned change! The motivation behind this was: given an ident, give me the corresponding entid if the ident names an attribute. `get_entid` isn't enough. `get_attribute` doesn't return the entid! But if we no longer use this, I'll kill it.

Reply:
Oh, and indeed it is — `attribute_for_ident` still has (and uses) this signature.